mobile/mobile.go

```diff
@@ -84,19 +88,15 @@ var (
 
 	jsonCBRegex = regexp.MustCompile(`(\w+)\.(\w+)\.(\w+)`)
 
-	m *mobileClient
+	m map[string]*mobileClient
```

Comment:
is this being initialised anywhere? 😅

Reply:
if not.. this will crash on access. Has this been tested?

Reply:
LOL good catch.

Wasn't sure you would be OK with this approach. Going to update the Android bindings in `lnc-rn` and test this end-to-end today.